includes/amp-helper-functions.php

```diff
@@ -929,9 +929,20 @@ function amp_get_content_sanitizers( $post = null ) {
 	}
 
 	if ( $amp_to_amp_linking_enabled ) {
-		$sanitizers['AMP_Link_Sanitizer'] = [
-			'paired' => ! amp_is_canonical(),
-		];
+
+		/**
+		 * Normally, when in a paired mode, links to the same origin will be to AMP.
+		 *
+		 * This allows passing URLs to exclude from having AMP-to-AMP links.
```

Comment:
Suggested docblock:

> Filters the list of URLs which are excluded from being included in AMP-to-AMP linking.
> 
> This only applies when the `amp_to_amp_linking_enabled` filter returns true,
> which it does by default in Transitional mode. This filter can be used to opt-in
> when in Reader mode. This does not apply in Standard mode. Only frontend
> URLs on the frontend need be excluded, as all other URLs are never made into AMP links.

Reply:
Sure, I committed this verbatim in https://github.com/ampproject/amp-wp/pull/4146/commits/77118d73956f7551eb4e108eb8150b1dc4585ab6, though the line breaks are slightly different.